src/components/Staking/StakingStatsBox.tsx

```diff
@@ -1,78 +1,71 @@
 // Import libraries
-import React, { useState, useEffect } from "react"
-import styled from "@emotion/styled"
+import React, { useState, useEffect, ReactNode } from "react"
 import { useIntl } from "react-intl"
-import { Spinner } from "@chakra-ui/react"
+import {
+  Code,
+  Divider,
+  Flex,
+  Icon,
+  Spinner,
+  useBreakpointValue,
+  VStack,
+} from "@chakra-ui/react"
 // Import components
 import Translation from "../Translation"
 import Tooltip from "../Tooltip"
 import Link from "../Link"
-import Icon from "../Icon"
 // Import utilities
 import { Lang } from "../../utils/languages"
 import { getData } from "../../utils/cache"
 import { getLocaleForNumberFormat } from "../../utils/translations"
+import { MdInfoOutline } from "react-icons/md"
 
 // Constants
 const NA_ERROR = "n/a"
 const ZERO = "0"
 const MAX_EFFECTIVE_BALANCE = 32
 
-// Styled components
-const Container = styled.div`
-  display: flex;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex-direction: column;
-  }
-`
-
-const Cell = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 1rem 2rem;
-  border-left: 1px solid ${({ theme }) => theme.colors.preBorder};
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    border-left: none;
-    border-top: 1px solid #33333355;
-  }
-  &:first-child {
-    border-left: none;
-    border-top: none;
-  }
-`
+const Cell: React.FC<{ children: ReactNode }> = ({ children }) => {
+  return (
+    <VStack spacing={2} py={4} px={8}>
```

Comment:
```suggestion
    <VStack 
      spacing={2} 
      py={4} 
      px={8}
      borderLeft={{ md: "1px" }}
      borderTop={{ base: "1px", md: "none" }}
      // `!important` needed to force an override of the user-agent
      borderColor="preBorder !important"
      _first={{
        borderLeft: "none",
        borderTop: "none",
      }}
    >
```
With the `!important` keyword needed for `borderColor`, not really sure why an underlying style from the user-agent is given specificity here.

`_first` is a pseudo prop for `:first-of-type` which works just the same here.